automation/tests/conftest.py

```diff
@@ -39,3 +39,21 @@ def mock_sendgrid_client():
         "sendgrid.SendGridAPIClient", autospec=True
     ) as mock_client, mock.patch("sendgrid.helpers.mail.Mail", autospec=True):
         yield mock_client
+
+
+@pytest.fixture
+def mock_auth0_client():
+    with mock.patch(
```

Comment:
I think you should create mocks of the `auth0.Auth0Client` object directly in the test with `mock.Mock(auth0.Auth0Client)`, rather than patching the constructor, that seems a little more straightforward.

Reply:
Example: https://github.com/bedstuystrong/automation_prime/pull/44/files#diff-3c7f183b3c0bf5016edf8df5b2e603d44bf69c10a454f13b51cc079a3c26f56aR27